### ISSUE TITLE
Negative Loss

### ISSUE THREAD

**Issue by VoiceBeer** [milesial/Pytorch-UNet#65]
Hi, thank you for your code. I am a student trying to use unet as a pre-trained model to do the segmentation part of my project. I'm now using my own data to train the model, but get negative loss like issue #7 . 

You mentioned making sure the target values are between 0 and 1, but I find that when loading the data, you have done the normalization to change the value to [0, 1] (the function _normalize_ in the file _util.py_ . So it's impossible that during the training, the values are not between 0 and 1 right? If so, why the loss becomes negative? Do you know how to fix it?

Looking forward to your reply and have a good day :)

**Comment by milesial** [comment]
Hi, I use the [BCE loss](https://pytorch.org/docs/stable/nn.html?highlight=bce#torch.nn.BCELoss) so you should refer there for any problems. The **targets** (here the true masks) should be between 0 and 1, and you should check that is the case for you.
The `normalize` function only applies to input images and not the targets.

**Comment by VoiceBeer** [comment]
@milesial I see, sry I misunderstood the meaning of targets, thank you for your reply!
